ui/package.json

```diff
@@ -132,12 +131,12 @@
     "zone.js": "0.8.26"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.13.10",
```

Comment:
this package has a strange versioning scheme.  "0.3.10" is also "v7-lts" and "0.803.29" is "v8-lts"